src/Service.Tests/dab-config.MsSql.json

```diff
@@ -1,868 +1,1832 @@
 {
-  "$schema": "../../schemas/dab.draft.schema.json",
+  "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
   "data-source": {
     "database-type": "mssql",
+    "connection-string": "Server=tcp:127.0.0.1,1433;Persist Security Info=False;User ID=sa;Password=REPLACEME;MultipleActiveResultSets=False;Connection Timeout=5;",
     "options": {
       "set-session-context": true
-    },
-    "connection-string": "Server=tcp:127.0.0.1,1433;Persist Security Info=False;User ID=sa;Password=REPLACEME;MultipleActiveResultSets=False;Connection Timeout=5;"
+    }
   },
   "runtime": {
     "rest": {
       "enabled": true,
       "path": "/api"
     },
     "graphql": {
-      "allow-introspection": true,
       "enabled": true,
-      "path": "/graphql"
+      "path": "/graphql",
+      "allow-introspection": true
     },
     "host": {
-      "mode": "development",
       "cors": {
-        "origins": [
-          "http://localhost:5000"
-        ],
+        "origins": ["http://localhost:5000"],
         "allow-credentials": false
       },
       "authentication": {
-        "provider": "StaticWebApps"
-      }
+        "provider": "StaticWebApps",
+        "jwt": {
```

Comment:
No, it's not required to be there, `AuthenticationOptions.Jwt` is a `null`able type, but it's probably more a case of how the `ConfigGenerator` in the CLI project works.

Take this line for example, from the `TryCreateRuntimeConfig` method:

```csharp
Authentication: new(options.AuthenticationProvider, new(options.Audience, options.Issuer)),
```

When this executes, it doesn't do anything to check if the `options.Audience` and `options.Issuer` are `null`, or that it's a SWA auth provider, so you'll potentially end up with the JSON as we see here.

This could be addressed in a few ways, we could make the CLI smarter in how it generates the config so possibly setting `null` for the `Jwt` parameters, or we could make a custom `JsonConverter` which only overrides the `Write` method and will drop the `Jwt` parameters based on conditions.

Ultimately, this happened because it's not smart enough to apply neuances to the object graph generated, and then we serialize as it was in memory.